internal/commands/app/describe.go

```diff
@@ -0,0 +1,65 @@
+package app
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/spf13/pflag"
+)
+
+// set of supported `app describe` command strings
+const (
+	CommandDescribeUse     = "describe"
```

Comment:
if these are simply only used in the `commands/commands.go` file, then we don't need consts for them (those consts only really became necessary when we were trying to print out suggested commands for any log messages)